BuildaUtils.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "BuildaUtils"
-  s.version      = "0.3.2"
+  s.version      = "0.4.2"
```

Comment:
Awesome! 🎉 

Do you mind doing those 2 fixes and let me know? 

I would be more than happy to merge this and then release a new version of `BuildaUtils` along with #14